docs/xks/operator-guide/kubernetes/eks.md

```diff
@@ -101,6 +101,133 @@ Run Terraform in the following order:
 Make sure that you only have one peering request open at the same time, else the accepter side won't be able to find a unique request.
 Now you should be able to see the VPC peering connected on both sides.
 
+## Update cluster version
+
+Updating EKS cluster version can not be done by updating Terraform code only, it also involves the AWS CLI and kubectl. Find your EKS version to upgrade to here: [EKS versions](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html)
+
+For further information on the AWS CLI commands used in this section, please refer to [AWS EKS CLI](https://docs.aws.amazon.com/cli/latest/reference/eks/index.html)
+
+### Update the control plane
+
+Get the name of the cluster to update:
+
+```bash
+aws eks list-clusters --region eu-west-1
+```
+
+Update the control plane version by running the following command with the actual values for \<cluster-name\> and \<version>\:
+
+```bash
+aws eks update-cluster-version --region eu-west-1 --name <cluster-name> --kubernetes-version <version>
+```
+
+The above command provides an id that can be use to check the status of the update:
+
+```bash
+aws eks describe-update --region eu-west-1 --name <cluster-name> --update-id <id>
+```
+
+The update is finished when status is `Successful`. Previous updates have approximately taken 45 minutes.
```

Comment:
Can you put `45 minutes` to make it super visible.
If I ever have to do this a 3am I don't want to get scared of how long it takes and it should be very clear.